packages/apollo-engine-reporting-protobuf/src/reports.proto

```diff
@@ -109,18 +119,65 @@ message Trace {
 		reserved 4;
 	}
 
+	// FIXME: document all new fields
+	message QueryPlanNode {
+		message SequenceNode {
+			repeated QueryPlanNode nodes = 1;
+		}
+		message ParallelNode {
+			repeated QueryPlanNode nodes = 1;
+		}
+		message FetchNode {
+			string serviceName = 1;
+			string operation = 2;
```

Comment:
Let's get rid of this until we build UI that needs it. In general we probably shouldn't collect any `string` data that we don't immediately present to users, to help them form a better model of what data we collect.

Reply:
Add a comment noting that we might want to augment it with more information about the operation at a later date when we build a feature.